### DISCUSSION TITLE
Wunderbaum vs FancyTree

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#6072] [General]
[FancyTree ](https://github.com/mar10/fancytree) (Which we use for our node tree) hasn't received any significant updates for over a year. This isn't too much of an issue since it mostly 'just works' for us, but considering that the FancyTree dev is working on a successor called [Wunderbaum](https://github.com/mar10/wunderbaum), we may want to look into migrating to that at some point if it reaches or surpasses FancyTree in feature parity.

**Comment by armync** [comment]
As far as it goes, it seems that it is consolidated and still worked on (as a successor). The wunderbaum demo is lacking some theming and forms but it has the trees. It is made on TypeScript.

Is it in a production-ready state for Trilium? @eliandoran 

[Wunderbaum Demo](https://mar10.github.io/wunderbaum/demo/#demo-welcome) and [FancyTree Demo](https://wwwendt.de/tech/fancytree/demo/#welcome.html)

**Reply by eliandoran** [reply]
@armync , I've considered Wunderbaum multiple times thus far.
Honestly, I'm a bit concerned.

It's still in beta (version zero point something) and it doesn't seem to be [truly actively developed](https://github.com/mar10/wunderbaum/graphs/contributors):

<img width="823" height="212" alt="image" src="https://github.com/user-attachments/assets/cace52be-9af5-496c-8c89-6b9e0a901bd0" />

As such, perhaps we should consider a different tree technology altogether.
